nudge.py

```
@@ -17,13 +18,17 @@
             time_difference = time.time() - float(words[-1])
         print(f"{words[0]} last seen {time_difference:.2f} seconds ago")
-    
-        # Nudge is named for the recipient, so the partner finds it under their own name.
+
+        # Nudge is named for the recipient, so the partner finds it under their own name. After message is sent, nudge_sent is True. If time difference < 5 seconds, nudge_sent is False.
         if time_difference > 5:
-            with open(f"mailroom/{partner}-nudge.txt", "w") as f:
-                f.write("hey you've been quiet")
-            print("hey you've been quiet")
-            
+            if nudge_sent == False:
```

Ruff (0.16.2)

[error] Avoid equality comparisons to `False`; use `not nudge_sent:` for false checks

Replace with `not nudge_sent`

(E712)